mxgraph-client/pom.xml

```diff
@@ -51,244 +49,197 @@
-                    <include>js/io/mxCodec.js</include>
-                    <include>js/io/mxObjectCodec.js</include>
-                    <include>js/io/mxCellCodec.js</include>
-                    <include>js/io/mxModelCodec.js</include>
-                    <include>js/io/mxRootChangeCodec.js</include>
-                    <include>js/io/mxChildChangeCodec.js</include>
-                    <include>js/io/mxTerminalChangeCodec.js</include>
-                    <include>js/io/mxGenericChangeCodec.js</include>
-                    <include>js/io/mxGraphCodec.js</include>
-                    <include>js/io/mxGraphViewCodec.js</include>
-                    <include>js/io/mxStylesheetCodec.js</include>
-                    <include>js/io/mxDefaultKeyHandlerCodec.js</include>
-                    <include>js/io/mxDefaultToolbarCodec.js</include>
-                    <include>js/io/mxDefaultPopupMenuCodec.js</include>
-                    <include>js/io/mxEditorCodec.js</include>
-                  </includes>
-                </aggregation>
-              </aggregations>
-            </configuration>
-          </execution>
-          <execution>
-            <!-- Minify the mxgraph client style sheets before packing the jar. -->
-            <id>minify-mxgraph-client-style</id>
+            <id>copy-mxgraph-client-resources</id>
             <phase>process-resources</phase>
             <goals>
-              <goal>compress</goal>
+              <goal>copy-resources</goal>
             </goals>
             <configuration>
-              <sourceDirectory>${mxgraph.path}/javascript/src</sourceDirectory>
-              <outputDirectory>${mxgraph.path}/javascript/src</outputDirectory>
-              <excludes>
-                <exclude>**/*.js</exclude>
-              </excludes>
-              <!-- Overwrite the original version with the minified one because some of the style sheets are loaded
-                automatically by the JavaScript code so we shouldn't change their name. -->
-              <nosuffix>true</nosuffix>
+              <outputDirectory>${webjar.contentDirectory}</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${mxgraph.path}/javascript/src</directory>
+                </resource>
+              </resources>
             </configuration>
           </execution>
         </executions>
       </plugin>
       <plugin>
-        <groupId>com.google.code.maven-replacer-plugin</groupId>
-        <artifactId>replacer</artifactId>
-        <version>1.5.3</version>
+        <groupId>com.github.blutorange</groupId>
+        <artifactId>closure-compiler-maven-plugin</artifactId>
         <executions>
           <execution>
-            <id>remove-mxClient-includes</id>
-            <phase>process-resources</phase>
-            <goals>
-              <goal>replace</goal>
-            </goals>
+            <id>default-minify-1</id>
```

Comment:
Would be good to add a comment that we're overwriting the default WebJar JavaScript "aggregation" step.